saleor/graphql/meta/mutations/utils.py

```diff
@@ -1,11 +1,13 @@
+from typing import Union
+
 from django.core.exceptions import FieldDoesNotExist, ValidationError
 from django.db import DatabaseError
 
 from ....core.error_codes import MetadataErrorCode
 
 
-def save_instance(instance, metadata_field: str):
-    fields = [metadata_field]
+def save_instance(instance, metadata_field: Union[str, list[str]]):
+    fields = [metadata_field] if isinstance(metadata_field, str) else metadata_field
```

Comment:
In my opinion, we should refactor this function so that it always receives a list of fields. 
Currently, `metadata_field` is not a clear name for the variable. The name suggests that we should pass one field but typing suggests it could be one field or a list of fields. It could be confusing in the future.

Reply:
right, naming is misleading - it would be better to pass a list in all cases